README.md

```diff
@@ -7,7 +7,11 @@
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 ```text
-Your answer...
+A Sql database relies on tables and it's inter relationship to one another. If you change one, you have to change all in matters such as deleting or altering. It relies on migrations to create the tables and to make changes to them. In a noSql database, everything can be changed within its own collection. When changes are made, it doesn't affect any others within the database and it doesn't require migrations to create or make changes to the database. It's as easy as going into the collection and making the edit. that's it!.
+
+You would use a sql db if you are creating a database that has a more uniformed structure like blog posts. every post will have an image, content, timestamp, and a unique id.
```

Comment:
also important to note that this would be used for something with a relationship. like posts/comments